jsapp/js/components/activity/activityMessage.component.tsx

```diff
@@ -0,0 +1,30 @@
+import Avatar from '../common/avatar';
+import {type ActivityLogsItem, AUDIT_ACTION_TYPES, FALLBACK_MESSAGE} from './activity.constants';
+import styles from './activityMessage.module.scss';
+
+/**
+ * An inline message that starts with avatar and username, and then is followed
+ * by short text describing what username did.
+ */
+export function ActivityMessage(props: {data: ActivityLogsItem}) {
```

Comment:
I personally prefer to destructure props so we would only have the data parameter and would get rid of the `props.`.